Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'sass-rails', '>= 6'
 gem 'webpacker', '~> 4.0'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem 'jbuilder', '~> 2.7'
+gem 'bootstrap', '~> 4.5.0'
```

Comment:
Since were using rails 6 had to have the latest bootstrap

Reply:
Getting fancy!